OgreMetalView.mm imports <SpriteKit/SpriteKit.h> but uses nothing from it
(only CAMetalLayer from QuartzCore). SpriteKit drags in SceneKit/GLKit, none
of which are part of the hermetic macOS SDK sysroot, so drop the import.

--- a/RenderSystems/Metal/src/Windowing/OSX/OgreMetalView.mm
+++ b/RenderSystems/Metal/src/Windowing/OSX/OgreMetalView.mm
@@ -28,7 +28,6 @@
 
 #import "OgreMetalView.h"
 
-#import <SpriteKit/SpriteKit.h>
 #import <QuartzCore/QuartzCore.h>
 
 @implementation OgreMetalView
